### DISCUSSION TITLE
getting error when the flopy.run_model() is runnig!

### DISCUSSION THREAD

**Discussion by zargol7419** [modflowpy/flopy#1741] [Q&A]
Hi everybody
I have used Flopy library to change the content of the well file and then run my existing Modflow model. I have changed the pumping rate of my wells based on a calculated matrix but when the model was running I got an error:
                                 MODFLOW-2000
   U.S. GEOLOGICAL SURVEY MODULAR FINITE-DIFFERENCE GROUND-WATER FLOW MODEL
                            Version 1.19.01 03/25/2010                      

Using NAME file: E:\zargolooo\Thesis\python-Thesis\4202-Transient-calib8_MODFLOW\Out_Mf2k 1\4202-Transient-calib8.mfn                                                                                                    
Run start date and time (yyyy/mm/dd hh:mm:ss): 2023/03/10 15:15:54

Error condition or warning reported

File "mf2kerr.p00" contains a description of the error or warning
here is the error and the content of mf2kerr:

 *****   SOLUTION DID NOT CONVERGE FOR HEADS   *****
 ***** FOR TIME STEP   1 IN STRESS PERIOD   72 *****

what does it mean? What is the problem with my model? I've got totally confused. Thanks in advance for sharing your knowledge and your answers.
I am from Iran so I have some problems with the internet connection I can not upload my screenshots so I just copied the content of errors!

**Comment by ccalvocm** [comment]
Greetings @zargol7419 :

MODFLOW-2000 shows convergence errors in the steady-state stress period due to inaccurate or inadequate parameters entered in the input file. This could be due to incorrect hydraulic conductivities (check LPF package), head boundaries (check CHD or GHB or whatsoever), or other parameter values. It may also be due to insufficient iteration (iterations must continue until convergence for steady-state solutions (check PCG, DE4, or what you are using)) or numerical instabilities associated with the numerical methods used by MODFLOW-2000. There might be water balance issues depending on your flux-dependent packages, so there's a lot to check

The model is being used for your thesis, if no confidentiality issues are involved I could check the model for you.
